devtools/buildVisualStories.js

```diff
@@ -0,0 +1,63 @@
+const { lstatSync, readdirSync, writeFileSync } = require('fs');
+const path = require('path');
+
+const srcPath = path.join(__dirname, '../src');
+
+const isComponentDirectory = (source) => {
+    const ignoredDirectories = ['utils', 'Docs'];
+    return lstatSync(source).isDirectory() && !ignoredDirectories.some(ignored => source.includes(ignored));
+};
+
+const componentDirs = readdirSync(srcPath).map(name => path.join(srcPath, name)).filter(isComponentDirectory).map(directory => {
+    return {
+        path: `${directory}/__stories__/`,
+        fileNames: readdirSync(`${directory}/__stories__/`)
+    };
+});
+
+// For every component directory.
+componentDirs.map((directory) => {
+    let fileContents = '';
+    let componentName = '';
+    // Loop through its files.
+    directory.fileNames.map((fileName) => {
+        // get only stories.js files
+        if (fileName.includes('.stories.js')) {
+        // Grab the component name
+            componentName = fileName.substr(0, fileName.indexOf('.'));
+            // TO DO: reenable storyshots for examples using hooks in storybook@6
+            // https://github.com/storybookjs/storybook/releases/tag/v6.0.0-alpha.43
+            if (componentName === 'Calendar'
+            || componentName === 'Dialog'
+            || componentName === 'Table'
+            || componentName === 'Popover'
+            || componentName === 'Shellbar') {
+                return;
+            }
+
+            fileContents += `
+import React from 'react';
+import * as stories from './${componentName}.stories';
+
+export default {
+    title: 'Visual'
+};
+
+export const ${componentName} = () => {
+    let storyNames = Object.keys(stories).filter(story => story !== 'default');
+
+    return (<>{storyNames.map(item => stories[item]())}</>);
```

Comment:
I was looking at the Token examples
<img width="897" alt="Screen Shot 2020-05-18 at 9 16 09 AM" src="https://user-images.githubusercontent.com/5314713/82223550-699db800-98e8-11ea-8a7f-21079a0f241d.png">

Do you think these could be wrapped in a `div` here so they show up on new lines?

(separate issue: that readonly token shouldn't say Compact...)

Reply:
Good idea! We also do need to do an audit and cleanup a lot of the examples for things like the labels

Reply:
I had to delete the visual snapshots
```
rm -r storybook-testing/__image_snapshots__
```
and recreate them for it to register these changes (because the base files didn't really change)